packages/terra-hookshot/src/Hookshot.jsx

```diff
@@ -186,17 +216,17 @@ class Hookshot extends React.Component {
   }
 
   enableListeners() {
-    const target = this.props.targetRef();
-    if (!target) {
+    const childElement = this.getTargetRef() || this.getBoundingRef();
```

Comment:
Why would would we use the bounding ref as the childElement fallback if target ref is undefined? Would we not use the target coordinates? I see below we grab the `scrollParents` it I would have though we'd still want to worry about the targeted scroll area

Reply:
It's an odd behavior, but one that I figured should work. If the user wants an element positioned according to screen coordinates, but also doesn't want it to leave a specific bounding element within the DOM flow, we still want to trap it.

Normally would be use the targetRef to find the elements to find scroll listeners from, but as we don't have an element to find parentNodes, we use the backup on the boundindRef as it could scroll as well. Technically I could see some wild scenario where the boundingRef and the targetRef aren't in the same flow and would need to merge the scroll listeners, but I figured we'll cross that bridge when we come to it.